### DISCUSSION TITLE
Closing tabs in compact sidebar form

### DISCUSSION THREAD

**Discussion by smzm** [zen-browser/desktop#2450] [Ideas]
For who using compact form of sidebar like picture below there is no quick way to close tabs. if you want to close any tab using  mouse you should **right click** on the tab and choose `close tab` from the menu. 

I think there should be a better and faster approach like press `ctrl` and click on tab to close, or when hover over website icon shows a close button.

![scrot-2024-10-31_Oct-10-1730358895_64x695_snip](https://github.com/user-attachments/assets/4b3cc585-5f43-4a07-b97c-cb42fb242846)

**Comment by Fireye04** [comment]
Came here to add this. Love the extra real-estate that compact tabs gives, but I need a keyboard shortcut or something to kill a tab i'm hovering. Solution proposed above works great as well.

Best I can do at the moment is either the workaround mentioned above or clicking then ctrl+w.

**Comment by smzm** [comment]
Thank you 
I think the compact form of showing tabs is the thing that makes a browser's interface super clean and focused. 👍
